libc/config/baremetal/arm/entrypoints.txt

```diff
@@ -247,6 +247,8 @@ set(TARGET_LIBM_ENTRYPOINTS
     libc.src.math.cos
     libc.src.math.cosf
     libc.src.math.coshf
+    libc.src.math.dfmal
+    libc.src.math.dsubl
     libc.src.math.erff
```

Comment:
I think you should remove the new entrypoints from all baremetal targets, not just baremetal RISC-V.